pkg/ee/default-application-catalog/applicationdefinitions/k8sgpt-app.yaml

```diff
@@ -34,6 +34,13 @@ spec:
             chartVersion: 0.1.0
             url: https://charts.k8sgpt.ai/
       version: v0.3.26
+    - template:
+        source:
+          helm:
+            chartName: k8sgpt-operator
+            chartVersion: 0.1.0
+            url: https://charts.k8sgpt.ai/
+        version: v0.3.48
```

Comment:
Where did you pick these versions from? From the official repo this should be the latest

https://github.com/k8sgpt-ai/charts/blob/main/k8sgpt-operator-0.2.6.tgz

Reply:
https://github.com/k8sgpt-ai/k8sgpt/blob/main/charts/k8sgpt/Chart.yaml is not `k8sgpt-operator` it's simply `k8sgpt`. The operator has a different project and thus helm chart  https://github.com/k8sgpt-ai/k8sgpt-operator/blob/main/chart/operator/Chart.yaml#L2